scss/components/sign-in-page.scss

```diff
@@ -0,0 +1,52 @@
+.not-authed {
+  min-height: 380px;
+  min-width: 476px;
+  padding: 10px;
+  display: flex;
+  align-items: center;
+  justify-content: space-evenly;
+  flex-direction: column;
+  position: relative;
+
+  img {
+    height: 128px;
+    width: 128px;
+  }
+
+  button {
+    border: none;
+    outline: none;
+    border-radius: 25px;
+    background-color: white;
+    color: $twitter-blue;
+    padding: 6px 16px 6px 6px;
+    font-weight: bold;
+    display: flex;
+    align-items: center;
+
+    img {
```

Comment:
This can now be removed

Reply:
img styles removed from sign-in-page.scss.